runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -192,6 +191,7 @@ def commonLegacyExcludeCategories = [
 ]
 
 def commonRunnerV2ExcludeCategories = [
+        'org.apache.beam.sdk.testing.UsesExternalService',
```

Comment:
Will this result in all Runner V2 test suites not running x-lang tests (essentially disabling all x-lang tests) ?

Reply:
Ah, thanks. We should fix the name in that case.